src/dotnet/APIView/APIView/APIView.csproj

```diff
@@ -14,6 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="MessagePack.Annotations" Version="3.0.54-alpha" />
```

Comment:
I don't see it used within APIView project. Do we really need it here in this csproj? If required, can we not use alpha version as dependency.

Reply:
Removed

Reply:
I still see it here. Did you push your change?